packages/core/src/components/Loader/Loader.types.ts

```diff
@@ -0,0 +1,19 @@
+export type LoaderColors = "primary" | "secondary" | "onPrimary" | "dark";
+
+export type LoaderSizes = "xs" | "small" | "medium" | "large";
+
+export type LoaderSize = LoaderSizes | number;
+
+export const mapSizesToLoaderSize: Record<LoaderSizes, number> = {
+  xs: 16,
+  small: 24,
+  medium: 40,
+  large: 64
+};
+
+export const mapLoaderColorsToColors: Record<LoaderColors, string> = {
+  primary: "primary-color",
+  secondary: "secondary-text-color",
+  onPrimary: "text-color-on-inverted",
+  dark: "primary-text-color"
+};
```

Comment:
Let's move these to the component maybe?

Reply:
Done